### ISSUE TITLE
Repeaterbook.com integration

### ISSUE THREAD

**Issue by joergschultzelutter** [joergschultzelutter/mpad#31]
* Prototype (based on CHIRPs Repeaterbook integration) works. Nevertheless, download of all country files is not possible due  to web server restrictions. 
* Have contacted RB admins on how to resolve the issue - if possible at all.

**Comment by 9M2PJU** [comment]
repeaterbook is better and it covers all repeaters around the world.

**Comment by joergschultzelutter** [comment]
Hi @9M2PJU 

Thanks for your comments. Most of the code for this particular integration is already done. Nevertheless, there are a few blockers which prevent me from moving forward:

- Unlike other sites like Hearham, the admins behind Repeaterbook are rather protective of their data. An API _does_ exist but comes with tons of caveats:
- There is no one-file-covers-everything download mechanism. I need to acquire the files per country per state.
- Due to the repeaterbook site's web server requirements, each download has to have a 10 mins pause adjacent to the next download in queue.
- However, even that is not enough - although my code sample is based on Chirp's [code](https://github.com/kk7ds/chirp/blob/master/chirp/sources/repeaterbook.py) (using its own user agent), I managed to get my updated MPAD test instance permanently blocked - which means that now, I cannot download _ANYTHING_ at all.
- I have been in touch with Gareth / repeaterbook.com, but to no avail (at least for now). Licensing an API key is out of the question for me as MPAD is (and will continue) as open source without any commercial elements in place - which is one  of the reasons why I had to remove the Openweathermap integration and replace that one with a free wx API. Additionally, everyone interested in running their own MPAD instance would need to undergo the very same procedure, too. My suggestion to Gareth was a model similar to aprs.fi's API usage account model, but so far I haven't heard back from him. Keep in mind that functionality like this would need to get developed, too.

In summary: unless we can find a solution that satisfies repeaterbook's legitimate interest in protecting their very own data AND also granting MPAD instances full-blown access to that data, I don't see any way of how we could establish that integration - at least for now. Nevertheless, I will keep this ticket open for the time being.

**Comment by 9M2PJU** [comment]
I have posted an issue, but it seems delete.
